src/remote/api_test.go

```diff
@@ -23,8 +23,8 @@ func TestHostnameNotFound(t *testing.T) {
 	token, err := api.GetToken("mytoken")
 	assert.Nil(t, token)
 	assert.Error(t, err)
-	assert.Contains(t, err.Error(), `Get "https://foo.bar.com/v2/storage/tokens/verify": dial tcp: lookup foo.bar.com: No address associated with hostname`)
-	assert.Regexp(t, `DEBUG  HTTP-ERROR\tGet "https://foo.bar.com/v2/storage/tokens/verify": dial tcp: lookup foo.bar.com: No address associated with hostname`, logs.String())
+	assert.Contains(t, err.Error(), `Get "https://foo.bar.com/v2/storage/tokens/verify": dial tcp`)
```

Comment:
Medzi jednotlivými verziami debianu/go sa tato hlaske lisi, preto som ju vyhodil.
(skusal som `alpine` base image, ale neprinieslo to ziadne vyhody).